pytest-extension/doc/changes/changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changes
 
 * [unreleased](unreleased.md)
+* [0.2.2](changes_0.2.2.md)
```

Comment:
I am assuming this intentionally has its own (and different) version number?

Reply:
Different from what, sorry?